### PR TITLE
[Merged by Bors] - fix go vet

### PR DESCRIPTION
**Pull request by noamnelke**
## Motivation
`go vet ./...` reports some errors.

## Changes
Fix `go vet` errors.
One issue was an actual bug where we'd call `continue` in a loop instead of performing the rest of the loop.
Deleted some zombie code as well.

### CODE REVIEW

File: activation/activation_test.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ var (
 	postProver       = &postProverClientMock{}
 	npst             = NewNIPSTWithChallenge(&chlng, poetRef)
 	commitment       = &types.PostProof{
-		//Identity:     []byte(nil),
+		// Identity:     []byte(nil),
```
Comment (antonlerner) [inline activation/activation_test.go]: remove commment?

File: activation/activationdb_test.go [hunk 2]
```diff
@@ -201,7 +172,7 @@ func TestATX_ActiveSetForLayerView(t *testing.T) {
 	rand.Seed(1234573298579)
 	atxdb, layers, _ := getAtxDb(t.Name())
 	blocksMap := make(map[types.BlockID]struct{})
-	//layers.AtxDB = &AtxDbMock{make(map[types.ATXID]*types.ActivationTx), make(map[types.ATXID]*types.NIPST)}
+	// layers.AtxDB = &AtxDbMock{make(map[types.ATXID]*types.ActivationTx), make(map[types.ATXID]*types.NIPST)}
```
Comment (antonlerner) [inline activation/activationdb_test.go]: remove comment?

File: activation/activationdb_test.go [hunk 3]
```diff
@@ -224,7 +195,7 @@ func TestATX_ActiveSetForLayerView(t *testing.T) {
 		hash, err := atx.NIPSTChallenge.Hash()
 		assert.NoError(t, err)
 		atx.Nipst = NewNIPSTWithChallenge(hash, poetRef)
-		//layers.AtxDB.(*AtxDbMock).AddAtx(atx.ID(), atx)
+		// layers.AtxDB.(*AtxDbMock).AddAtx(atx.ID(), atx)
```
Comment (antonlerner) [inline activation/activationdb_test.go]: same?

File: miner/builder_test.go [hunk 4]
```diff
@@ -556,7 +535,7 @@ func TestBlockBuilder_getVotes(t *testing.T) {
 	beginRound := make(chan types.LayerID)
 	n1 := service.NewSimulator().NewNode()
 	allblocks := []*types.Block{b1, b2, b3, b4, b5, b6, b7}
-	//st := []types.BlockID{b1.ID(), b2.ID(), b3.ID()}
+	// st := []types.BlockID{b1.ID(), b2.ID(), b3.ID()}
```
Comment (antonlerner) [inline miner/builder_test.go]: remove?

File: miner/builder_test.go [hunk 5]
```diff
@@ -129,7 +108,7 @@ func TestBlockBuilder_StartStop(t *testing.T) {
 	net := service.NewSimulator()
 	beginRound := make(chan types.LayerID)
 	n := net.NewNode()
-	//receiver := net.NewNode()
+	// receiver := net.NewNode()
```
Comment (antonlerner) [inline miner/builder_test.go]: remove?

File: activation/activation_test.go (hunk 1 above)
Comment (noamnelke) [reply activation/activation_test.go]: Sure, I just used the IDE's auto-format, didn't actually look at every line, will remove all of these now.

**Review by antonlerner — APPROVED** [review]

**Review by y0sher — APPROVED** [review]
you changed some value receivers to be pointer receivers was that a `go vet` thing ?

**Comment by noamnelke** [comment]
> you changed some value receivers to be pointer receivers was that a `go vet` thing ?

Yes, it warns that we pass a mutex by value.

e.g.:
```
activation/atxdb.go:570: getTopAtx passes lock by value: activation.ActivationDb
```

**Comment by noamnelke** [comment]
bors merge

**Comment by y0sher** [comment]
ok. though we could also change the mutex to be of type pointer..

**Comment by noamnelke** [comment]
Yes, but it's a bit more involved. A pointer to a mutex needs to be initialized (it starts out being `nil`), while making the method receiver a pointer is more straightforward.

I wonder why you resist this... Is it because using a pointer receiver implies that the method mutates the struct? In all of the places that I changed all other methods, except the one I changed, had pointer receivers, so it's not like the struct is supposed to be immutable.

**Comment by noamnelke** [comment]
bors merge

**Pull request closed without merging**